### DISCUSSION TITLE
Deployment Error

### DISCUSSION THREAD

**Discussion by Boschgopi** [juspay/hyperswitch#4101] [Q&A]
```
git clone https://github.com/juspay/hyperswitch-cdk.git
cd hyperswitch-cdk
bash install.sh

```

This API will be removed in the next major release.
⏳ Bootstrapping environment aws://767397921485/us-east-1...
Trusted accounts for deployment: (none)
Trusted accounts for lookup: (none)
Using default execution policy of 'arn:aws:iam::aws:policy/AdministratorAccess'. Pass '--cloudformation-execution-policies' to customize.
CDKToolkit: creating CloudFormation changeset...
❌ Environment aws://767397921485/us-east-1 failed bootstrapping: Error [ValidationError]: Stack:arn:aws:cloudformation:us-east-1:767397921485:stack/CDKToolkit/a4990440-e1f6-11ee-88da-0e853effafe9 is in DELETE_FAILED state and can not be updated.
at Request.extractError (/usr/lib/node_modules/aws-cdk/lib/index.js:379:46692)
at Request.callListeners (/usr/lib/node_modules/aws-cdk/lib/index.js:379:91437)
at Request.emit (/usr/lib/node_modules/aws-cdk/lib/index.js:379:90885)
at Request.emit (/usr/lib/node_modules/aws-cdk/lib/index.js:379:199281)
at Request.transition (/usr/lib/node_modules/aws-cdk/lib/index.js:379:192833)
at AcceptorStateMachine.runTo (/usr/lib/node_modules/aws-cdk/lib/index.js:379:157705)
at /usr/lib/node_modules/aws-cdk/lib/index.js:379:158035
at Request. (/usr/lib/node_modules/aws-cdk/lib/index.js:379:193125)
at Request. (/usr/lib/node_modules/aws-cdk/lib/index.js:379:199356)
at Request.callListeners (/usr/lib/node_modules/aws-cdk/lib/index.js:379:91605) {
code: 'ValidationError',
time: 2024-03-15T06:46:20.524Z,
requestId: 'ed28bb08-98da-4797-8468-a960a885026d',
statusCode: 400,
retryable: false,
retryDelay: 686.9833624271799
}
Stack:arn:aws:cloudformation:us-east-1:767397921485:stack/CDKToolkit/a4990440-e1f6-11ee-88da-0e853effafe9 is in DELETE_FAILED state and can not be updated.
An error occurred (NoSuchBucket) when calling the ListObjectVersions operation: The specified bucket does not exist
An error occurred (NoSuchBucket) when calling the ListObjectVersions operation: The specified bucket does not exist
An error occurred (InvalidClientTokenId) when calling the ListAttachedRolePolicies operation: The security token included in the request is invalid

**Comment by HyperSwitchers** [comment]
@Boschgopi The error message is `Stack:arn:aws:cloudformation:us-east-1:767397921485:stack/CDKToolkit/a4990440-e1f6-11ee-88da-0e853effafe9 is in DELETE_FAILED state and can not be updated`. To fix it you need to go to **AWS console dashboard > CloudFormation >  Stacks** and then select **CDKToolkit** and delete it manually. Now after deleting `CDKToolkit` try to re-run the script with same steps you mentioned

**Reply by Boschgopi** [reply]
CDKToolkit: creating CloudFormation changeset...

2:45:59 pm | CREATE_FAILED        | AWS::IAM::Role          | LookupRole

The security token included in the request is invalid (Service: AmazonIdentityManagement; Status Code: 403; Error Code: InvalidClientTokenId; Request ID: d384cd7d-c941-4ca1-a9f1-e1ffff28cd66; Proxy: nul

l)

**Reply by jagan-jaya** [reply]
@Boschgopi Please make sure that your AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY and AWS_SESSION_TOKEN are valid and the user you are using has Administrator access

**Reply by Boschgopi** [reply]
I used the valid credentials and user also having the administrator access, But still getting same error.

**Reply by HyperSwitchers** [reply]
@Boschgopi can you share the full error log if possible?
